numberformat/diff.emu

```diff
@@ -1017,11 +1072,13 @@
             1. Let _str_ be *"-0"*.
           1. Else,
             1. Let _str_ be ! Number::toString(_x_).
-        1. If the grammar cannot interpret _str_ as an expansion of |StringNumericLiteral|, return *NaN*.
-        1. Let _mv_ be the MV, a mathematical value, of ? ToNumber(_str_), as described in <emu-xref href="#sec-runtime-semantics-mv-s"></emu-xref>.
+        1. Let _text_ be ! StringToCodePoints(_str_).
+        1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
+        1. If _literal_ is a List of errors, let _mv_ be *NaN*.
+        1. Else, let _mv_ be the StringIntlMV of _literal_.
         1. If _mv_ is 0 and the first non white space code point in _str_ is `-`, return *-0*.
-        1. If _mv_ is 10<sup>10000</sup> and _str_ contains `Infinity`, return *+&infin;*.
-        1. If _mv_ is -10<sup>10000</sup> and _str_ contains `Infinity`, return *-&infin;*.
+        1. <del>If _mv_ is 10<sup>10000</sup> and _str_ contains `Infinity`, return *+&infin;*.</del>
```

Comment:
why do we need a &lt;del&gt; here ? ToIntlMathematicalValue is a new AO in this proposal,  &lt;del&gt; should only apply to delection on pre-existing ECMA402 or ECMA262 steps.

Reply:
Thanks for pointing to that,  it's already corrected